modules/util/svg/svg.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package svg
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/microcosm-cc/bluemonday"
+
+	minify "github.com/tdewolff/minify/v2"
+	"github.com/tdewolff/minify/v2/svg"
+)
+
+// MinifySVG compact svg strings
+func MinifySVG(svgData io.Reader) (*bytes.Buffer, error) {
```

Comment:
Since package name has `svg`. The `function` name could be simple as `Minify`.

Reply:
Or probably better `Sanitize`

Reply:
@lafriks there is a sanitize function. The main goal of minify is to be able to use the same test fixture from know lib as the indentation of bluemonday is not the same. Maybe we would kept it to serve rendered/filtered file but it is not necessary.